spec/support/blueprints.rb

```diff
@@ -97,6 +97,7 @@ def self.sn
   device { device_messages.first.device || Device.make }
   institution { device.institution || Institution.make }
   sample_identifier { SampleIdentifier.make(sample: Sample.make(institution: object.institution, patient: object.patient, encounter: object.encounter)) }
+  # sample_identifier { SampleIdentifier.make(entity_id: "sample-#{Sham.sn}", sample: Sample.make(institution: device.institution)) }
```

Comment:
This comment just slipped in or it _will_ have sense in the future but not right now?